ethstorage/p2p/protocol/syncclient.go

```diff
@@ -689,6 +700,7 @@ func (s *SyncClient) assignBlobRangeTasks() {
 				s.OnBlobsByRange(res)
 			}(pr.id)
 		}
+		t.lastSubTaskIdx = nextIdx - 1
```

Comment:
The same concern as line 425. Also can we remove this line by moving down line 630?